lambda_functions/automate_s2_nbar_rolling_archive/execute_s2nbar/handler.py

```diff
@@ -0,0 +1,38 @@
+import logging
```

Comment:
This PR doesn't need an extra lambda function, it should just use one of the existing generic ones.